src/main/java/user_interface/custom_game/custom_game_screens/CustomGameEditorScreen.java

```diff
@@ -0,0 +1,52 @@
+package user_interface.custom_game.custom_game_screens;
+
+import javax.swing.*;
+import java.awt.*;
+
+class CustomGameEditorScreen implements ICustomGameScreen {
```

Comment:
Since we changed our classes from Screens to Panels, it might be better if you rename the classes in terms of panels.